client/sites-dashboard/components/sites-site-url.ts

```diff
@@ -1,14 +1,25 @@
 import styled from '@emotion/styled';
+import { ExternalLink } from '@wordpress/components';
 
-export const SiteUrl = styled.a`
+export const SiteUrl = styled( ExternalLink )`
```

Comment:
I think we should move the `margin-right: 4px;` from the `Truncated` component to this extension of `ExternalLink`.

It makes more sense as we can style the icon directly instead of styling the truncated text: we want to shift the icon to the left, not to add more right space to the truncated text. So something like:

```css
.components-external-link__icon {
    margin-left: 4px;
}
```

Would make sense. And then removing the `margin` rule from the `Truncated` component.